pyproject.toml

```diff
@@ -1,8 +1,3 @@
-[tool.black]
-line-length = 120
-skip-string-normalization = true
-target-version = ['py38']
-
 [tool.pylint.main]
```

Comment:
While not strictly related to this PR, can you remove this?

Reply:
I tried to merge it with the ruff configuration where possible and removed the rest